brion/plugin/spikeReportBinary.cpp

```diff
@@ -159,25 +132,22 @@ bool SpikeReportBinary::handles( const SpikeReportInitData& initData )
 
 std::string SpikeReportBinary::getDescription()
 {
-    return "Blue Brain binary spike reports:\n"
-           "  [file://]/path/to/report" +
-           std::string( BINARY_REPORT_FILE_EXT );
-}
-void SpikeReportBinary::close()
-{
+    return "Blue Brain binary spike reports:  "
+           "[file://]/path/to/report" + std::string( BINARY_REPORT_FILE_EXT );
 }
 
 Spikes SpikeReportBinary::read( const float )
 {
     // In file based reports, this function reads all remaining data.
     Spikes spikes;
-    Spike* spikeArray = _memFile.getSpikes();
-    const size_t nElems = _memFile.getSpikeCount();
+    const Spike* spikeArray = _memFile->getReadableSpikes();
+    const size_t nElems = _memFile->getNSpikes();
+    _currentTime = spikeArray[nElems - 1].first +
```

Comment:
In the case of an empty spike report file (only a header, with no spikes, this will access to an empty array